core/shared/src/main/scala/scalaz/zio/internal/NamedThreadFactory.scala

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018 - 2019 John A. De Goes. All rights reserved.
+package scalaz.zio.internal
+
+import java.util.concurrent._
+import java.util.concurrent.atomic.AtomicInteger
+
+final class NamedThreadFactory(name: String, daemon: Boolean) extends ThreadFactory {
```

Comment:
No chance to keep this open? 
What if I want to make use of: https://github.com/OpenHFT/Java-Thread-Affinity ?